install/management/appliance.go

```diff
@@ -301,18 +279,22 @@ func (d *Dispatcher) createAppliance(conf *configuration.Configuration) error {
 			Value: fmt.Sprintf("-serveraddr=0.0.0.0 -port=%s -port-layer-port=8080 %s", d.DockerPort, d.dockertlsargs)})
 		spec.ExtraConfig = append(spec.ExtraConfig, &types.OptionValue{Key: "guestinfo.vch/sbin/vicadmin",
 			Value: fmt.Sprintf("-docker-host=unix:///var/run/docker.sock -insecure -sdk=%s -ds=%s -vm-path=%s -cluster=%s -pool=%s %s",
-				conf.TargetPath, conf.ImageStorePath, vm.InventoryPath, conf.ClusterPath, conf.ResourcePoolPath, vicadmintlsargs)})
+				conf.Target, conf.ImageStores[0], vm.InventoryPath, conf.ClusterPath, d.vchPoolPath, vicadmintlsargs)})
 	} else {
 		d.VICAdminProto = "http"
 		d.DockerPort = "2375"
 		spec.ExtraConfig = append(spec.ExtraConfig, &types.OptionValue{Key: "guestinfo.vch/sbin/docker-engine-server",
 			Value: fmt.Sprintf("-serveraddr=0.0.0.0 -port=%s -port-layer-port=8080", d.DockerPort)})
 		spec.ExtraConfig = append(spec.ExtraConfig, &types.OptionValue{Key: "guestinfo.vch/sbin/vicadmin",
 			Value: fmt.Sprintf("-docker-host=unix:///var/run/docker.sock -insecure -sdk=%s -ds=%s -vm-path=%s -cluster=%s -pool=%s -tls=%t",
-				conf.TargetPath, conf.ImageStorePath, vm.InventoryPath, conf.ClusterPath, conf.ResourcePoolPath, false)})
+				conf.Target, conf.ImageStores[0], vm.InventoryPath, conf.ClusterPath, d.vchPoolPath, false)})
 	}
 	spec.ExtraConfig = append(spec.ExtraConfig, &types.OptionValue{Key: "guestinfo.vch/files", Value: files})
 
+	cfg := make(map[string]string)
+	extraconfig.EncodeWithPrefix(extraconfig.MapSink(cfg), conf, "vch")
```

Comment:
What I'm thinking is the conf will change for VM reconfiguration, but right now, there is nothing happened in VCH, so will remove this duplication.